lib/mintacoin/accounts/crypto/stellar.ex

```diff
@@ -4,47 +4,101 @@ defmodule Mintacoin.Accounts.Stellar do
   """
 
   alias Mintacoin.Accounts.Crypto.AccountResponse
-  alias Stellar.{Horizon, Horizon.Transaction, KeyPair, TxBuild}
 
-  @type status :: :ok | :error
-  @type stellar_response :: map()
+  alias Stellar.{
+    Horizon,
+    Horizon.Transaction,
+    KeyPair,
+    TxBuild,
+    TxBuild.CreateAccount,
+    TxBuild.SequenceNumber,
+    TxBuild.Signature
+  }
```

Comment:
@Odraxs separate this in another alias like this:
```suggestion
  alias Stellar.TxBuild.{CreateAccount, SequenceNumber, Signature}
```